com.github.d4nj1.tlpui.desktop

```diff
@@ -0,0 +1,14 @@
+[Desktop Entry]
```

Comment:
This file is at https://github.com/d4nj1/TLPUI/blob/master/tlpui.desktop and doesn't need to be here.

Reply:
Also since you are upstream the desktop file is invalid. `Icon` MUST be an icon name not a path. `Exec` should be an executable name, possible with arguments, not a path.

Reply:
If I remember correctly there were some issues when building Debian packages when not defined like this. I check if this is still the case.

* Regarding Exec:
It is allowed to have the full path but I will try to only mention the executable directly as you desire.
See https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-1.1.html#exec-variables 

* Regarding Icon:
Same as for Exec the full path can be used. I used it because I was unsure if the icons that are defined and applied to the current theme engine in Tlpui get recognized by Flatpak.
See https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-1.1.html#recognized-keys

I try to adopt to one `.desktop` file and update the PR accordingly.